internal/cmd/tui.go

```diff
@@ -37,7 +37,8 @@ func tuiCmd() *cobra.Command {
 				return err
 			}
 
-			blocks, err := loadTasks(proj, cmd.OutOrStdout(), cmd.InOrStdin(), false)
+			loader := getLoader(cmd)
```

Comment:
It would be fine if `project.NewLoader` would have a return with `error` as well.

Reply:
@pastuxso can you push a commit on top with the changes you have in mind?